Automattic/wp-calypso
# Pricing Grid: Fix action buttons

#### Proposed Changes

* This PR is one among several stacked PRs that implements the new pricing grid design. In https://github.com/Automattic/wp-calypso/pull/71372, we introduced a new signup flow /start/onboarding-2023-pricing-grid that is only available in dev and wpcalypso environments.
* The design for the pricing grid can be found in 2e991-pb. Please keep the Figma design open in a separate tab.
* The parent PR is https://github.com/Automattic/wp-calypso/pull/71410.
* This PR adds styling changes for the plan CTAs and also makes the free plan and enterprise plan CTAs functional.

<img width="1502" alt="Screenshot 2022-12-26 at 12 34 58 PM" src="https://user-images.githubusercontent.com/1269602/209515831-2166b18d-9760-4c7e-9fde-6cb9c332044e.png">


#### What Not To Test

* Responsive screens. It will be implemented in a separate PR. Right now, test in a full width desktop browser viewport.
* Non-en. The first launch will be `en` only.
* Verify UI changes only in the plan header section (everything up to the CTA buttons). The plan features will be styled in subsequent PRs.



#### Testing Instructions



* Visit `/start/onboarding-2023-pricing-grid`.
* Verify that the CTA styling matches the Figma. In particular pay attention to fonts and paddings. 
* Verify that clicking each plan behaves appropriately - free plan should create a free site, premium plan should take to checkout with a premium plan in cart etc.
* Verify tracks events for each of the above are appropriately fired. Particularly, the tracks events for free plan should match the one in production. 
* Verify that clicking "Get in touch" on the Enterprise plan opens the VIP landing page in a new tab. Verify that the external link tracking event is fired. No other plan related Tracks event should fire.
* Verify that the production `onboarding` flow at `/start` is unaffected by UI changes.
